config.go

```diff
@@ -119,6 +121,11 @@ func init() {
 		defaultOneShot = false
 	}
 
+	defaultMetricsPort, err := strconv.ParseUint(getEnv("VAULT_METRICS_PORT", "8080"), 10, 16)
+	if err != nil {
+		defaultOneShot = false
```

Comment:
What is the purpose of this line?

Reply:
This should be `defaultMetrics port = 8080`. I disagree with the way this default behaviour is done for all of the flags, but this is consistent at least.